src/IO/WebSocket/WslinkImageStream/index.js

```diff
@@ -0,0 +1,158 @@
+/* global Blob window URL */
+import CompositeClosureHelper from '../../../Common/Core/CompositeClosureHelper';
+
+function wslinkImageStream(publicAPI, model) {
+  if (!model.client) throw Error('Client must be provided');
+
+  model.metadata = null;
+  model.activeURL = null;
+  model.fps = 0;
+  model.lastTime = +(new Date());
+  model.view_id = -1;
```

Comment:
It's good to have fresh eyes! I made view_id have a default value, and it will be set when the publicAPI.connect() method is called. 

It doesn't make sense to change it in the middle of a session, does it?